lib/escape.js

```diff
@@ -53,6 +53,20 @@ exports.escapeHtml = function (input) {
 };
 
 
+exports.escapeJson = function (input) {
+
+    if (!input) {
+        return '';
+    }
+
+    return input.replace(/</g, '\\u003c')
```

Comment:
If we wanted to squeak out a tiny bit more performance we could utilize char codes for the comparison as well. It's not a drastic improvement like we saw with the previous switch (only ~5% improvement) and I'm not sure if it fits the style of the rest of the library but figured I'd suggest it if we wanted to do it.

I put together some benchmarks here to show an example: https://jsperf.com/string-replacement-methods/1